AssemblyAiConsole.csproj

```diff
@@ -3,8 +3,5 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Ignore this. I opted out of implicit usings so the code would be more backwards compatible for older version of .NET that don't have implicit usings.